tests/core/test_bungiesearch.py

```diff
@@ -300,8 +300,7 @@ def test_optimal_queries(self):
         db_item = NoUpdatedField.objects.get(pk=1)
         src_item = NoUpdatedField.objects.search.query('match', field_title='My title')[0]
         self.assertEqual(src_item.id, db_item.id, 'Searching for the object did not return the expected object id.')
-        self.assertTrue(src_item._meta.proxy, 'Was expecting a proxy model after fetching item.')
-        self.assertEqual(src_item._meta.proxy_for_model, NoUpdatedField, 'Proxy for model of search item is not "NoUpdatedField".')
+        self.assertEqual(src_item.get_deferred_fields(), {'field_description'}, 'Was expecting description in the set of deferred fields.')
```

Comment:
yes. It looks like older versions of Django created a new proxy model every time you executed a queryset with `.only`. Django 1.10+ just uses the same model and sets the fields differently.

The intent of this test seemed to be ensuring that `.only` was used in the Django queryset, so this test now tests that more directly and in a backwards compatible way.